packages/std/Cargo.toml

```diff
@@ -29,6 +29,8 @@ staking = []
 # stargate enables stargate-dependent messages and queries, like raw protobuf messages
 # as well as ibc-related functionality
 stargate = []
+# eureka enables eureka-dependent messages and queries
+eureka = []
```

Comment:
Could you also add it to the docs.rs features further up?

Reply:
Done